### PR TITLE
Feature/gamejam command module

### PR DESCRIPTION
**Pull request by Banananaman91**
In progress command module seeking code review

### CODE REVIEW

File: backend/AdjutantBot/Modules/GameJamModule.cs [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                    case findGameJamResponse.Success:
+                        await ReplyAsync($"Created team {team.Team.TeamName}. Team Leader is {Context.User.Mention}. Game jam ID is {team.Team.GameJamId}.");
+                        break;
+                    case findGameJamResponse.Failure:
+                        await ReplyAsync("Sorry, there are no Game Jams to join");
+                        break;
+                }
+                
+            }
+            else await ReplyAsync("You must give a team name.");
```
Comment (RubyNova) [inline backend/AdjutantBot/Modules/GameJamModule.cs]: Add braces please.

File: backend/AdjutantBot/Modules/GameJamModule.cs [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                    case AddTeamMemberModificationResponse.AlreadyMember:
+                        await ReplyAsync($"You are already a member of {team.Team.TeamName}");
+                        break;
+                    case AddTeamMemberModificationResponse.FullMembership:
+                        await ReplyAsync(
+                            $"{team.Team.TeamName} is already full on members. Please contact the Team Leader {Context.Client.GetUser(team.Team.TeamCaptainId).Mention} if this is a mistake");
+                        break;
+                }
+            }
+            else await ReplyAsync("You must give a team name.");
```
Comment (RubyNova) [inline backend/AdjutantBot/Modules/GameJamModule.cs]: Add braces please.

File: backend/AdjutantBot/Modules/GameJamModule.cs [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+                        await ReplyAsync(
+                            $"You are not authorized to perform that action. Only {Context.Client.GetUser(team.Team.TeamCaptainId).Mention} can change the team size");
+                        break;
+                    case SetLimitModifierResponse.Overlimit:
+                        await ReplyAsync($"You have requested a team size greater than the limit authorized by this game jam.");
+                        break;
+                }
+            }
+
+            if (teamName == null) await ReplyAsync("You need to input a team name");
```
Comment (RubyNova) [inline backend/AdjutantBot/Modules/GameJamModule.cs]: Please change from `==` to `is`.

File: backend/AdjutantBot/Models/GameJamResponse.cs [hunk 4]
```diff
@@ -0,0 +1,8 @@
+namespace AdjutantBot.Models
+{
+    public enum findGameJamResponse
```
[the remark this answers is not in the document]
Comment (Inzanit) [reply backend/AdjutantBot/Models/GameJamResponse.cs]: Looks like this is already resolved

**Pull request merged**